template-skeleton/template/src/components/views/Main.js

```diff
@@ -205,24 +203,15 @@ const useStylesMapContainer = makeStyles((theme) => ({
     left: theme.spacing(4),
     zIndex: 1,
 
-    [theme.breakpoints.down('sm')]: {
-      bottom: theme.spacing(4),
-      left: theme.spacing(2),
+    [theme.breakpoints.down('xs')]: {
+      display: 'none',
     },
   },
   cartoLogoMap: {
```

Comment:
Why is this class defined here? Don't be better to move this to CartoLogoMap component?

Reply:
It's not CartoLogoMap, it's ZoomControl. It's always better define if a children is shown or not from parent, so the children haven't the context. This code is in sample-app.

Reply:
Done